tests/program/test_extract.py

```diff
@@ -200,3 +200,101 @@ def test_program_extract_dialect_keys_option():
     assert json.loads(result.stdout) == extract(
         path=file, dialect={"keys": ["name", "id"]}
     )
+
+
+def test_program_extract_valid_rows_1004():
+    result = runner.invoke(program, "extract data/countries.csv --valid")
+    assert result.exit_code == 0
+    assert (
+        result.stdout.count("1  Ireland      Britain  67")
+        and result.stdout.count("3  22           Germany  83")
```

Comment:
This test (and similar) might be really fragile regarding formatting. It will be hard to maintain also. Nothing prevents us to use something like `extract data/countries.csv --valid --csv/json` to get an output in another format. For example, getting a JSON output and using `json.parse` to get a Python dict